src/screens/WalletConnect/WalletConnectList.js

```diff
@@ -49,7 +50,7 @@ const style = StyleSheet.create({
     flexDirection: 'row',
     alignItems: 'center',
     borderBottomWidth: 1,
-    borderBottomColor: '#cecece',
+    borderBottomColor: COLORS.borderColor,
```

Comment:
Changed color from `#cecece` to `#eeeeee`

Reply:
Why?

Reply:
As discussed, this is close enough to the current border color to compose the app theme.

Reply:
Reverted to original color on [2e29c07](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/327/commits/2e29c070354d1fae1ee605c2884a2c6ff80cb0b2)